components/dashboard/src/start/CreateWorkspace.tsx

```diff
@@ -284,22 +284,27 @@ function RepositoryNotFoundView(p: { error: StartWorkspaceError }) {
 
       if (!updatedRecently) {
         setStatusMessage(<div className="mt-2 flex flex-col space-y-8">
-          <p className="text-base text-gray-400 w-96">Permission to access private repositories has been granted. If you are a member of '{owner}', please try to request access for Gitpod.</p>
+          <p className="text-base text-gray-400 w-96">Permission to access private repositories has been granted. If you are a member of <code>{owner}</code>, please try to request access for Gitpod.</p>
           <a className="mx-auto" href={authorizeURL}><button className="secondary">Request Access for Gitpod</button></a>
         </div>);
         return;
       }
 
       setStatusMessage(<div className="mt-2 flex flex-col space-y-8">
-        <p className="text-base text-gray-400 w-96">Your access token was updated recently. Please try again if the repository exists and Gitpod was approved for '{owner}'.</p>
+        <p className="text-base text-gray-400 w-96">Your access token was updated recently. Please try again if the repository exists and Gitpod was approved for <code>{owner}</code>.</p>
         <a className="mx-auto" href={authorizeURL}><button className="secondary">Try Again</button></a>
       </div>);
     })();
   }, []);
 
-  return <StartPage phase={StartPhase.Checking} error={p.error}>
-    {statusMessage}
-  </StartPage>;
+  return (
+    <StartPage phase={StartPhase.Checking} error={p.error}>
+      <p className="text-base text-gitpod-red mt-2">
+        <code>{repoFullName}</code>
```

Comment:
Thanks! I didn't know about `CodeText`, and it looked promising at first... but it's a `span` whereas I would rather use a `code` for this.

What's the intended use of `CodeText`? Why would I want to have a styled `span` that looks like a styled `code` instead of using a styled `code`?
For example, those variable names in `Account.tsx` should be marked up with a `code`.

So now I need advice on how to handle this in the best interest of the project 😊:

- [ ] Use `CodeText` as-is (🙈 I'd rather not contribute more non-semantic markup)
- [ ] Leave `code` here but copy-paste the style from `CodeText` (🤕 repetition over encapsulation)
- [ ] Modify `CodeText` to be a `code` instead of a `span` (🤔 but are there any cases in the codebase where a span is actually the right element?)
- [ ] Make `CodeText` into a polymorphic component via a classic `as` property, defaulting to `span` for retro-compatibility (🖌 might be too much, given it's unlikely anyone would ever want `CodeText` to render as a `section`, or a `dl`, or anything other than a `span` or a `code`.

Reply:
🍊 **Answering some questions:**

> Why would I want to have a styled span that looks like a styled code instead of using a styled code?

@trumbitta Using `<code>` tag to render such elements sounds right, especially since this is broadly related to rendering markdown elements like _code blocks_ and _inline code_. The idea behind `CodeText` component was to wrap such elements and provide proper styling when rendering instances across the dashboard.

> For example, those variable names in `Account.tsx` should be marked up with a code.

@trumbitta You're right and this can also help with accessibility and component consistency. I'm also open for discussing a different design between inline code like the variables in `/account` and the repository full name in the error pages here. 💯 

**🍋  Some background context:**

FWIW, for historical reasons, the `CodeText` component was introduced as a side fix during a larger feature implementation in https://github.com/gitpod-io/gitpod/pull/4607, to replace the already existing span-based elements from the initial dashboard redesign. To help cut some corners and unblock shipping that larger feature, we had to limit [scope creep](https://en.wikipedia.org/wiki/Scope_creep) by limiting improvements related to these code elements to just componetizing the already existing elements. 🔖 

FYI, we're approaching the development of the dashboard using an MVC (Minimum Viable Change) approach and we iteratively improve existing or add new functionalities. ➿ 

**🥝 Proposal:**

1. Taking all the above into account, in the spirit of MVC and iterating step-by-step as product needs grow, I'd suggest to keep the `CodeText` component as is for now. If you could use `CodeText` for replacing the repository full name below that would be great. Otherwise, feel free to leave this as is.
2. Next steps could include improving this `CodeText` component by using the `<code>` tag under the hood instead of a `<span>`, while checking if this renders ok across the dashboard.

Reply:
💯 Makes sense.
Thanks for the bit of history and context explanation: now I feel more aligned with the approach and guiding principles 🎛️ . It will be easier to contribute and think strategically 🧐 !

Balancing MVC (first time I see this acronym with a glorious past used with this meaning 😸 ) with my old core principle of continuous refactoring (here's a fun take by Kent Beck: https://docs.google.com/document/d/e/2PACX-1vRS232prL-Bq44hmmdzJTHkCz-nYBlQVPAtemOmVfDP1rbiD777jQAaUpIeF2KCdSSEJMEwHTeo9QdE/pub) promises to be a refreshing and positive challenge 🙇 so thank you for that 🙏 

🪡 I'll update the bits you pointed out to use `CodeText`, and I will open an issue for improving it with the `<code>` tag.

Reply:
Thanks a lot again @gtsiolis for the detailed explanation! ❤️

Reply:
Thanks @Siddhant-K-code @trumbitta! 🙏 💯

Reply:
@gtsiolis done!